### PR TITLE
javascript/secret-handshake: add minimal notes

### PR DESCRIPTION
**Pull request by SleeplessByte**
Add minimal mentor notes for `secret-handshake`.

### CODE REVIEW

File: tracks/javascript/exercises/secret-handshake/mentoring.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  if (!(typeof sekret === 'number')) {
+    throw new Error('Handshake must be a number')
+  }
+
+  const shake = OPERATIONS.filter((_, i) => sekret & (1 << i))
+  return sekret & 16 ? shake.reverse() : shake
+}
+```
+
+Note that the `secret & 16` can also be written as `1 << 4` or `0b10000` or
```
Comment (tejasbubane) [inline tracks/javascript/exercises/secret-handshake/mentoring.md]: `sekret` or `secret` ?

File: tracks/javascript/exercises/secret-handshake/mentoring.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+  const bits = sekret.toString(2).split('').reverse()
+  const shake = OPERATIONS.filter((_, i) => bits[i] === '1')
+  return bits[4] === '1' ? shake.reverse() : shake
+}
+```
+
+### Common suggestions
+
+- If a student checks for each value explicitely (as listed in the comments of
```
Comment (tejasbubane) [inline tracks/javascript/exercises/secret-handshake/mentoring.md]: `*explicitly`

File: tracks/javascript/exercises/secret-handshake/mentoring.md (hunk 1 above)
Comment (SleeplessByte) [reply tracks/javascript/exercises/secret-handshake/mentoring.md]: Ah good catch. Thank you! It was sekret on purpose indeed

**Comment by rpottsoh** [comment]
GitHub thinks Travis CI build is in progress.
image

However if I examine the [Details](https://travis-ci.org/exercism/website-copy/builds/500291630?utm_source=github_status&utm_medium=notification) the check appears to have passed:

image

I presume rebasing onto master will trigger Travis CI to perform another check.

**Comment by SleeplessByte** [comment]
@rpottsoh I've done exactly that 🗡

**Pull request merged**